README.md

```diff
@@ -106,3 +114,7 @@ To unmount the filesystem:
 ```
 fusermount -u <mount_point>
 ```
+or on MacOS:
+```
+diskutil unmount <mount_point>
```

Comment:
I've been using `umount <mount_point>`, does that work for you too? If so it's probably preferable since it's shorter.

Reply:
I did find `umount`, but its man page says
> **NOTES**
Due to the complex and interwoven nature of Mac OS X, umount may fail often.  It is recommended that diskutil(1) (as in, `diskutil unmount /mnt`) be used
     instead.

so I've been using `diskutil` instead. I'm happy to go with either though!

Reply:
Ah ok makes sense 👍